pypit/arspecobj.py

```diff
@@ -140,7 +144,7 @@ def init_exp(slf, scidx, det, fitsdict, trc_img, ypos=0.5, **kwargs):
         # Object traces
         if trc_img[sl]['nobj'] != 0:
             # Loop on objects
-            for qq in range(trc_img[sl]['traces'].shape[1]):
+            for qq in range(trc_img[sl]['nobj']):
```

Comment:
I think this needs to be reinstated to the original text (for the echelle pipeline). Was it broken?

Reply:
I have returned it to as it was.
I don't think it was broken.